### ISSUE TITLE
Issue launching the game

### ISSUE THREAD

**Issue by martsrzx91** [Disaczar/KetsInfiniteTechnologies#1]
![image](https://github.com/user-attachments/assets/782994a0-9d0a-483f-9ce0-0c8652905195)

**Comment by Deonisseyy** [comment]
I test it on space age and it works correct.
Can you provide modlist that you use?

**Comment by martsrzx91** [comment]
I could, but i am running pretty extensive mod list >_> I am 90% certain that it was compability issue with this mod and its dependacies: https://mods.factorio.com/mod/xiuxian
![image](https://github.com/user-attachments/assets/486d79fa-1c57-499a-8134-ac7e7666d4fd)

[mod-list.json](https://github.com/user-attachments/files/17705196/mod-list.json)

**Comment by martsrzx91** [comment]
Are you planning to fix this?

**Comment by Deonisseyy** [comment]
Yes. I will fix it and update this repository for Factorio 2.0. I plan to do it on this week.

**Comment by Deonisseyy** [comment]
I fixed the main problem with order.
But for your modlist requires special lab with all science packs inputs. I try to do this.
